entity-framework/core/miscellaneous/testing/sqlite.md

```diff
@@ -43,4 +43,8 @@ The simplest way to enable testing against a different database is to modify you
 
 The key to testing with this provider is the ability to tell the context to use SQLite, and control the scope of the in-memory database. The scope of the database is controlled by opening and closing the connection. The database is scoped to the duration that the connection is open. Typically you want a clean database for each test method.
 
+>[!TIP]
+> To use the `.SqliteConnection()` extension method, reference the NuGet package `Microsoft.Data.Sqlite`.
```

Comment:
There is no SqliteConnection extension method. What you see in the snippets is calling the constructor of SqliteConnection. Also, the Microsoft.Data.Sqlite package is included when you add the Microsoft.EntityFrameworkCore.Sqlite package. No need to add it explicitly.

Reply:
I agree it's not an extension method xD Copy&paste Documentation mistake . 
I just wanted to add the tip to share with the reader quickly a necessary package